### DISCUSSION TITLE
Random crashing not sure why.

### DISCUSSION THREAD

**Discussion by morpheus2n2** [Taxel/PlexTraktSync#1967] [Support] [answered]
I can't understand what's going on, I got it all setup and it works fine for a while but then the container will crash, problem is when I try and restart it will keep crashing, but rebuilding the container allows it to work for about an hour just fine as far as i can tell. 
https://pastebin.com/a4en19vm

**Comment by glensc** [comment, marked as answer]
1. crashes must be reported as issues
2. do not use external sites (especially sites with ads), attach logs as .txt file uploads
